test/resources/debug-darwin.txt

```diff
@@ -1,41 +1,7 @@
-npm init
----------
-This utility will walk you through creating a package.json file.
-It only covers the most common items, and tries to guess sensible defaults.
-
-See `npm help json` for definitive documentation on these fields
-and exactly what they do.
-
-Use `npm install <pkg> --save` afterwards to install a package and
-save it as a dependency in the package.json file.
-
-Press ^C at any time to quit.
-name: (test-suppose) awesome_package
-version: (1.0.0) 0.0.1
-description: It's an awesome package man!
-entry point: (index.js) 
-test command: npm test
-git repository: 
-keywords: awesome, cool
-author: JP Richardson
-license: (ISC) MIT
-About to write to /private/tmp/test-suppose/package.json:
-
-{
-  "name": "awesome_package",
-  "version": "0.0.1",
-  "description": "It's an awesome package man!",
-  "main": "index.js",
-  "scripts": {
-    "test": "npm test"
-  },
-  "keywords": [
-    "awesome",
-    "cool"
-  ],
-  "author": "JP Richardson",
-  "license": "MIT"
-}
-
-
-Is this ok? (yes) yes
+node /Users/stevendvachon/Desktop/node-suppose/test/helpers/cli-prompt.js
```

Comment:
Can't accept that, execution path must be relative, if not tests will never pass in CI.